src/mathdistops/qexp.py

```diff
@@ -1,4 +1,7 @@
-def qexp(p, rate, plot_graph=False):
```

Comment:
I would suggest, we rename the argument to `graph` and set it by default to True, such that it is consistent with the other functions. I think it is a good idea to add a default value for rate; e.g. 'rate=1'

Reply:
What do u think @carinaya, as you created the other function qexp ?

Reply:
we can do that, and if you do so, please let me know your final function expression. And I can update the readme file afterward

Reply:
Hi Sandy, Thank you for your comment. Can you pls try again? I modified qexp fucntion and test function both. On my local, p=1 cases passed, and coverge is 100%